src/ncp/ncp_base.cpp

```diff
@@ -1791,6 +1791,10 @@ otError NcpBase::GetPropertyHandler_CAPS(void)
     SuccessOrExit(error = mEncoder.WriteUintPacked(SPINEL_CAP_COUNTERS));
     SuccessOrExit(error = mEncoder.WriteUintPacked(SPINEL_CAP_UNSOL_UPDATE_FILTER));
 
+#if OPENTHREAD_CONFIG_NCP_ENABLE_POWER_STATE_SAVE
```

Comment:
Should this be OPENTHREAD_CONFIG_NCP_ENABLE_POWER_STATE_CONTROL?

Reply:
Indeed. Thanks.